### PR TITLE
chore(docs): update naming to match new branding initiative

### PR DESCRIPTION
**Pull request by jherey**
# Description
This PR follows the rebranding initiative by the Marketing and Docs team to change commercetools to Composable Commerce.

https://github.com/commercetools/clients-team-internal/issues/87

NB: This PR does not update the tests, as this is out of scope.

### CODE REVIEW

**Review by barbara79 — APPROVED** [review]
LGTM 👍

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "commercetools/commercetools-sdk",
   "license": "MIT",
   "type": "library",
-  "description": "The official PHP SDK to the commercetools eCommerce APIs",
+  "description": "The official PHP SDK of commercetools Composable Commerce APIs",
```
Comment (melindafekete) [inline composer.json:5]:
```suggestion
  "description": "The official PHP SDK of commercetools Composable Commerce APIs",
```
```suggestion
  "description": "The official PHP SDK for the commercetools Composable Commerce APIs",
```

File: lib/commercetools-api/docs/RequestBuilder.md [hunk 2]
```diff
@@ -2087,7 +2087,7 @@ $request = $builder
 
 Creates an order from a Cart from a specific Store.
 When using this endpoint the orders's store field is always set to the store specified in the path parameter.
-The cart must have a shipping address set before creating an order. When using the Platform TaxMode,
+The cart must have a shipping address set before creating an order. When using Composable Commerce TaxMode,
```
Comment (melindafekete) [inline lib/commercetools-api/docs/RequestBuilder.md:2090]:
```suggestion
The cart must have a shipping address set before creating an order. When using the Platform TaxMode,
```

File: lib/commercetools-api/docs/RequestBuilder.md [hunk 3]
```diff
@@ -3181,7 +3181,7 @@ $request = $builder
 
 Creates an order from a Cart.
 The cart must have a shipping address set before creating an order.
-When using the Platform TaxMode, the shipping address is used for tax calculation.
+When using Composable Commerce TaxMode, the shipping address is used for tax calculation.
```
Comment (melindafekete) [inline lib/commercetools-api/docs/RequestBuilder.md:3184]:
```suggestion
When using Platforme TaxMode, the shipping address is used for tax calculation.
```

File: lib/commercetools-api/src/Models/Cart/Cart.php [hunk 4]
```diff
@@ -144,7 +144,7 @@ public function getTotalPrice();
 
     /**
      * <p>Not set until the shipping address is set.
-     * Will be set automatically in the <code>Platform</code> TaxMode.
+     * Will be set automatically in <code>Composable Commerce</code> TaxMode.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/Cart.php:147]:
```suggestion
     * Will be set automatically in the <code>Platform</code> TaxMode.
```

File: lib/commercetools-api/src/Models/Cart/CartBuilder.php [hunk 5]
```diff
@@ -345,7 +345,7 @@ public function getTotalPrice()
 
     /**
      * <p>Not set until the shipping address is set.
-     * Will be set automatically in the <code>Platform</code> TaxMode.
+     * Will be set automatically in <code>Composable Commerce</code> TaxMode.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/CartBuilder.php:348]:
```suggestion
     * Will be set automatically in the <code>Platform</code> TaxMode.
```

File: lib/commercetools-api/src/Models/Cart/CartModel.php [hunk 6]
```diff
@@ -560,7 +560,7 @@ public function getTotalPrice()
 
     /**
      * <p>Not set until the shipping address is set.
-     * Will be set automatically in the <code>Platform</code> TaxMode.
+     * Will be set automatically in <code>Composable Commerce</code> TaxMode.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/CartModel.php:563]:
```suggestion
     * Will be set automatically in the <code>Platform</code> TaxMode.
```

File: lib/commercetools-api/src/Models/Cart/CustomLineItem.php [hunk 7]
```diff
@@ -98,7 +98,7 @@ public function getState();
     public function getTaxCategory();
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/CustomLineItem.php:101]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/CustomLineItemBuilder.php [hunk 8]
```diff
@@ -188,7 +188,7 @@ public function getTaxCategory()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/CustomLineItemBuilder.php:191]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/CustomLineItemModel.php [hunk 9]
```diff
@@ -307,7 +307,7 @@ public function getTaxCategory()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/CustomLineItemModel.php:310]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/LineItem.php [hunk 10]
```diff
@@ -143,7 +143,7 @@ public function getAddedAt();
     public function getState();
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/LineItem.php:146]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/LineItemBuilder.php [hunk 11]
```diff
@@ -284,7 +284,7 @@ public function getState()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/LineItemBuilder.php:287]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/LineItemModel.php [hunk 12]
```diff
@@ -463,7 +463,7 @@ public function getState()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/LineItemModel.php:466]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/ShippingInfo.php [hunk 13]
```diff
@@ -57,7 +57,7 @@ public function getShippingRate();
     public function getTaxedPrice();
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/ShippingInfo.php:60]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/ShippingInfoBuilder.php [hunk 14]
```diff
@@ -120,7 +120,7 @@ public function getTaxedPrice()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/ShippingInfoBuilder.php:123]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

File: lib/commercetools-api/src/Models/Cart/ShippingInfoModel.php [hunk 15]
```diff
@@ -186,7 +186,7 @@ public function getTaxedPrice()
     }
 
     /**
-     * <p>Will be set automatically in the <code>Platform</code> TaxMode once the shipping address is set is set.
+     * <p>Will be set automatically in <code>Composable Commerce</code> TaxMode once the shipping address is set is set.
```
Comment (melindafekete) [inline lib/commercetools-api/src/Models/Cart/ShippingInfoModel.php:189]:
```suggestion
     * <p>Will be set automatically in the <code>Platform</code> TaxMode, once the shipping address is set is set.
```

**Review by melindafekete — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (melindafekete) [inline composer.json:5]:
```suggestion
  "description": "The official PHP SDK for the commercetools Composable Commerce APIs",
```

**Comment by melindafekete** [comment]
@barbara79 @jherey do you understand why we can't rename Platform in 'Platform Taxmode', or shall I explain in further detail? Not sure if there's any more SDKs to do, but please keep an eye out for this one

**Comment by jenschude** [comment]
> @barbara79 @jherey do you understand why we can't rename Platform in 'Platform Taxmode', or shall I explain in further detail? Not sure if there's any more SDKs to do, but please keep an eye out for this one

I also think that the comment is not complete here. It's not talking about the platform itself but about the platform enum value.

**Comment by jherey** [comment]
> @barbara79 @jherey do you understand why we can't rename Platform in 'Platform Taxmode', or shall I explain in further detail? Not sure if there's any more SDKs to do, but please keep an eye out for this one

Oh yeah I do understand. It's mostly a group update using the code editor, I'll take time to check these updates going forward.

File: lib/commercetools-api/docs/RequestBuilder.md [hunk 16]
```diff
@@ -3181,7 +3181,7 @@ $request = $builder
 
 Creates an order from a Cart.
 The cart must have a shipping address set before creating an order.
-When using the Platform TaxMode, the shipping address is used for tax calculation.
+When using Platforme TaxMode, the shipping address is used for tax calculation.
```
Comment (melindafekete) [inline lib/commercetools-api/docs/RequestBuilder.md:3184]:
```suggestion
When using Platform TaxMode, the shipping address is used for tax calculation.
```

**Review by melindafekete — APPROVED** [review]

**Pull request merged**